launch/mapper_pensa.launch

```diff
@@ -1,20 +1,20 @@
 <launch>
 
-  <arg name="robot_frame_id"    default="/c0001_bl" />
-  <arg name="inertial_frame_id" default="store" />
+  <arg name="robot_base_link_frame_id" default="/c0001_bl" />
+  <arg name="inertial_frame_id"        default="lp0" />
```

Comment:
same here - if we can start getting rid of `lp0`s now, it will help us transition to multi-perch in the future